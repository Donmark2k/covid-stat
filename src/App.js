import { Route, Routes } from 'react-router-dom';
import NavBar from './components/navbar';
import Countries from './components/country';
import CountryDetails from './components/countryDetails';
import 'bootstrap/dist/css/bootstrap.min.css';

function App() {
  return (
    <div className="App">
      <NavBar />
      <Routes>
        <Route path="/" element={<Countries />} />
        <Route path="/:cityID" element={<CountryDetails />} />
      </Routes>
    </div>
  );
}

export default App;
