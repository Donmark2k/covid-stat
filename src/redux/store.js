// import { configureStore } from '@reduxjs/toolkit';
// import { createLogger } from 'redux-logger';
// import rocketReducer from './rockets/rocketSlice';
// import missionReducer from './missions/missionSlice';

// const logger = createLogger({
//   // Options for the logger can be passed here
//   // Show only the actions in the console
//   predicate: (getState, action) => action.type !== 'SOME_ACTION',
//   // Collapse the logs by default
//   collapsed: true,
// });

// export const store = configureStore({
//   reducer: {
//     missions: missionReducer,
//     rockets: rocketReducer,
//   },
//   middleware: (getDefaultMiddleware) => getDefaultMiddleware().concat(logger),
// });

// export default store;
