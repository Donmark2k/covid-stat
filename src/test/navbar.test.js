import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../redux/store';
import NavBar from '../components/navbar';
import App from '../App';
import { BrowserRouter } from 'react-router-dom';



it('Navbar page renders correctly', () => {
  const tree = renderer
    .create(
      <React.StrictMode>
      <Provider store={store}>
        <BrowserRouter>
          <NavBar />
        </BrowserRouter>
      </Provider>
    </React.StrictMode>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it('App renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <App />
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});