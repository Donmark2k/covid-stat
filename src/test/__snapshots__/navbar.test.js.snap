// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App  page renders correctly 1`] = `
<div>
  <h2>
    Something went wrong:
  </h2>
  <p>
    targetWindow.matchMedia is not a function
  </p>
  <button
    onClick={[Function]}
    type="button"
  >
    Try again
  </button>
</div>
`;

exports[`Navbar page renders correctly 1`] = `
<div>
  <h2>
    Something went wrong:
  </h2>
  <p>
    targetWindow.matchMedia is not a function
  </p>
  <button
    onClick={[Function]}
    type="button"
  >
    Try again
  </button>
</div>
`;
